res/controllers/lib/mididispatcher.mjs

```diff
@@ -0,0 +1,26 @@
+export class MidiDispatcher {
```

Comment:
Can you document the class and it’s members using JSdoc comments?

Reply:
Please review the documentation: 98a3d82